model-optimizer/extensions/back/CropToStridedSlice.py

```diff
@@ -102,9 +112,8 @@ def replace_pattern(self, graph: Graph, match: [str, Node]):
 
         source = node.in_port(0).get_connection().get_source()
 
-        stride = Const(graph, {'value': np.ones(shape_rank, dtype=np.int64)}).create_node()
-        ss = StridedSlice(graph, {'name': 'Crop_', 'begin_mask': begin_mask, 'end_mask': end_mask, 'new_axis_mask': np.array([0]),
-                                  'shrink_axis_mask': np.array([0]), 'ellipsis_mask': np.array([0])}).create_node()
+        stride = Const(graph, {'value': np.ones(shape_rank, dtype=np.int64),
```

Comment:
The same actions are done in all `if` and `elif` statements. So we can just calculate the `begin`, `end` values (not Node objects!) in these conditions and after them just create a SS node using `create_op_with_const_inputs`. So we reduce the amount of code significantly!

Reply:
Please, create unit tests for all three cases. Make sure that tests pass before the code refactoring and after it.

Reply:
I think best decision is not refactor this transformation, because in `if` and `elif` statements we done not same actions and created `Node` objects used both inside and outside these statements.

Reply:
Ok. Leave it as is